lib/widgets/itemsView.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
```

Comment:
Use this in the ListView builder here- https://github.com/shrkamat/ros/pull/1/files#diff-3ffa8ed17a5cbec6797cf9d8e357fb53e15e7197ea8100cd2041dfccfe2ec1a6R48